pySDC/implementations/problem_classes/RayleighBenard3D.py

```diff
@@ -354,3 +353,55 @@ def compute_Nusselt_numbers(self, u):
             't': Nusselt_t,
             'b': Nusselt_b,
         }
+
+    def get_frequency_spectrum(self, u):
```

Comment:
Could you add just a bit of documentation here ? One can understand from the function how it works, but it would be better to allow that directly from the docstring